libcontainer/container_linux.go

```diff
@@ -43,6 +43,7 @@ type linuxContainer struct {
 	criuVersion          int
 	state                containerState
 	created              time.Time
+	criuFeat             *criurpc.CriuFeatures
```

Comment:
criuFeat isn't a container property, so I think we can have a global variable for it. What do you think about this idea?